recipe/patches/0004-loosen-requirements.patch

```diff
@@ -1,5 +1,14 @@
+From 5fbf80906ab7cd1bfd429f40fb4ef95cd8ff370d Mon Sep 17 00:00:00 2001
+From: "Uwe L. Korn" <uwe.korn@quantco.com>
+Date: Tue, 6 Apr 2021 17:52:09 +0200
```

Comment:
I like having this meta-information in patches; taken from the commits with which you added the "header-less" patches.